app/javascript/mastodon/features/ui/components/disabled_account_banner.jsx

```diff
@@ -49,8 +49,6 @@ class DisabledAccountBanner extends PureComponent {
     e.stopPropagation();
 
     this.props.onLogout();
-
-    return false;
```

Comment:
`e.preventDefault()`; and `e.stopPropagation();` is doing the job of the return. the false is dead code.